assets/collapsible-content.css

```diff
@@ -41,8 +41,8 @@
 
 .collapsible-content-wrapper-narrow {
   margin: 0 auto;
-  padding-right: 1.5rem;
   padding-left: 1.5rem;
+  padding-right: 1.5rem;
```

Comment:
I don't think these affect functionality but maybe it's still worth it to change them to `-inline-start` and `-end` for consistency?